apps/analytics/urls.py

```diff
@@ -40,4 +40,9 @@
         views.get_last_submission_datetime_analysis,
         name="get_last_submission_datetime_analysis",
     ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/download_all_participants$",
```

Comment:
`challenge/(?P<challenge_pk>[0-9]+)/download_all_participants$` -> `challenges/(?P<challenge_pk>[0-9]+)/download_all_participants/$`. 
Please read https://restfulapi.net/resource-naming/ for API naming conventions.

Reply:
Hey @vkartik97, I think you missed this. Please change `challenge` -> `challenges` as mentioned above.